odoo/tools/safe_eval.py

```diff
@@ -324,16 +745,44 @@ def safe_eval(expr, globals_dict=None, locals_dict=None, mode="eval", nocopy=Fal
     This can be used to e.g. evaluate
     an OpenERP domain expression from an untrusted source.
 
+    :param expr: the expression to evaluate
+
+    :param globals_dict: a dictionary of global variables to use
+
+    :param locals_dict: a dictionary of local variables to use
+
+    :param mode: the mode in which to evaluate the expression
+                            (see the built-in function eval() for details)
+
+    :param nocopy: if False, the globals and locals dictionaries are
```

Comment:
comment marked as solved, but the `nocopy` argument still exists, we should remove it